options/locale/locale_en-US.ini

```
@@ -2525,6 +2525,7 @@ form.create_org_not_allowed = You are not allowed to create an organization.
 settings = Settings
 settings.options = Organization
 settings.full_name = Full Name
+settings.email = Contact EMail
```

```suggestion
settings.email = E-Mail
```

I see `E-Mail -> EMail` was a deliberate change, but I think the `E-Mail` spelling is much more common, and the I think the "Contact" wording is unnecessary.

---

Actually, with this change, we don't even need a new translation key. You should use the top-level `email` one:

https://github.com/go-gitea/gitea/blob/90b3b3dbf83dd88aab00d35535fe913b8d313bfb/options/locale/locale_en-US.ini#L29

---

Nope, see the wiki above

And gitea code always use email or Email of E-mail, you can grep the code.

---

https://en.wikipedia.org/wiki/Email

---

In any case, the linked translation is correct and should be used.

---

I think we should keep the Contact part. On other parts the E-Mail is used to e.g. allow getting Notifications or reset the Password. The "Contact" here indicates that this is only used on the Profile and nowhere else.

---

Imho, a Email is always used to "contact" a entity, there is no other purpose of it.

---

Gitea uses the User Mail to e.g. send Notification or reset the Password. It is also used together with git. This is not the case with the Organisation Mail. It is only a field that is shown on the Profile. It is not even verified. I think we should make this more clear, or Users expect that the can do the same with an Organisation Mail as with the User Mail.

---

I think both options are acceptable, but which other usecase is an organization email supposed to have? There is no password to reset and no notifications to send

---

Either `contact_email = Contact Email` or existing `email` is fine to me. Note that both are top-level and not inside teh `org` ini section because these terms are not org-specific.

---

```suggestion
settings.email = Contact Email
```